package.json

```diff
@@ -4,15 +4,28 @@
   "description": "A specialised child process spawn for `git` commands",
   "main": "gitexec.js",
   "scripts": {
-    "test": "node test.js"
+    "lint": "standard",
+    "unit": "node test.js",
+    "test": "npm run lint && npm run unit"
   },
-  "author": "Rod <rod@vagg.org> (http://r.va.gg/)",
+  "author": "Rod <rod@vagg.org> (https://r.va.gg/)",
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/rvagg/gitexec.git"
+    "url": "git+https://github.com/rvagg/gitexec.git"
   },
+  "bugs": {
+    "url": "https://github.com/rvagg/gitexec/issues"
+  },
+  "homepage": "https://github.com/rvagg/gitexec#readme",
+  "keywords": [],
   "dependencies": {
     "bl": "~1.0.0"
-  }
+  },
+  "devDependencies": {
+    "standard": "^14.3.1"
+  },
+  "files": [
```

Comment:
I don't really like `"file"`, I don't even use .npmignore much. If it doesn't get ignored by .gitignore then I'm fine with it going into a package. A package should be relatively complete, and ideally you should even be able to run tests from what you get in a package.

Reply:
I like files and it's needed otherwise your test file ends up in the package.

Reply:
As for the tests, if one wants that they should clone the repo

Reply:
I like test files in my packages, unless I have large fixtures that would bloat it, otherwise the overhead is minimal and packages are stand-alone

Reply:
It's just good practice. Imagine everyone thinking like that, which many do, then we end up with thousands of useless files.

Reply:
It's not good practice, it's just a practice that a sub-community of npm developers have adopted; there's a good many of us who disagree with it.
My preference for managing size is to manage dependency-creep. Developers complaining about test files but being OK with having dependency trees of over 2K packages is the most laughable symptom of this. Node.js pushes modularity to an extreme, but it doesn't mean you should throw caution to the wind.

Reply:
I definitely pay attention to my deps, and I don't like useless files.

That being said, I don't really mind for this project. It's just the standard thing to do for me.

If it helps to move forward faster, I can revert the files addition.

Reply:
FYI

```
C:\Users\xmr\Desktop\gitexec>npm pack --dry
npm notice
npm notice package: gitexec@1.0.0
npm notice === Tarball Contents ===
npm notice 861B  gitexec.js
npm notice 886B  test.js
npm notice 698B  package.json
npm notice 1.1kB LICENSE.md
npm notice 1.3kB README.md
npm notice 2.7kB gitexec-1.0.0.tgz
npm notice 728B  .github/workflows/test.yml
npm notice === Tarball Details ===
npm notice name:          gitexec
npm notice version:       1.0.0
npm notice filename:      gitexec-1.0.0.tgz
npm notice package size:  5.8 kB
npm notice unpacked size: 8.3 kB
npm notice shasum:        20caa288ef8d9f00e9c191f34dcbddc7db880afb
npm notice integrity:     sha512-3D+jonfdd5gii[...]n8uTTBAHZscDQ==
npm notice total files:   7
npm notice
gitexec-1.0.0.tgz
```

So, as you can see, more files are added like the CI files.

Reply:
I dropped the files change from here so that we move forward faster.